Src/Support/Google.Apis.Auth.Tests/OAuth2/DefaultCredentialProviderTests.cs

```diff
@@ -212,6 +212,32 @@ public class DefaultCredentialProviderTests
     ""regional_cred_verification_url"": ""https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15"",
     ""imdsv2_session_token_url"": ""http://169.254.169.254/latest/api/token""
   }}";
+        private const string DummyImpersonatedServiceAccountCredential = @"{
+""service_account_impersonation_url"": ""https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/service-account-email:generateAccessToken"",
+""delegates"": [
+  ""delegate-email-1"",
+  ""delegate-email-2""
+],
+""source_credentials"": {
+  ""client_id"": ""CLIENT_ID"",
+  ""client_secret"": ""CLIENT_SECRET"",
+  ""refresh_token"": ""REFRESH_TOKEN"",
+  ""type"": ""authorized_user""
+},
+""type"": ""impersonated_service_account""}";
+        private const string RecursiveImpersonatedServiceAccountCredential = @"{
```

Comment:
Possibly add a blank line between each of these service credentials? (Or better yet, maybe now's a good time to move them to resource files that we load? That way we don't need the quote doubling etc.)